ux.symfony.com/importmap.php

```diff
@@ -156,4 +168,20 @@
         'version' => '1.6.1',
         'type' => 'css',
     ],
+    'scheduler' => [
+        'version' => '0.23.0',
+    ],
+    'path-to-regexp' => [
+        'version' => '6.2.1',
+    ],
+    '@swup/theme' => [
+        'version' => '2.1.0',
+    ],
+    '@kurkle/color' => [
+        'version' => '0.3.2',
+    ],
+    'bootstrap/dist/css/bootstrap.min.css' => [
+        'version' => '5.3.2',
+        'type' => 'css',
+    ],
```

Comment:
Where did these new imports come from? Were they missing before? Or needed after upgrades?

Reply:
Good catch i'll check...

Reply:
Fixed (i must have made something i should not have 😅 )